common/src/main/scala/common/validation/Validation.scala

```diff
@@ -10,18 +10,18 @@ import cats.syntax.validated._
 import common.Checked
 import common.exception.AggregatedMessageException
 import common.validation.ErrorOr.ErrorOr
-import eu.timepit.refined.numeric.GreaterEqual
 import eu.timepit.refined._
 import eu.timepit.refined.api.Refined
+import eu.timepit.refined.numeric.Interval.Closed
 import org.slf4j.Logger
 
 import scala.concurrent.Future
 import scala.util.{Failure, Success, Try}
 
 object Validation {
 
-  type GreaterEqualOne = GreaterEqual[W.`1.0`.T]
-  type GreaterEqualRefined = Refined[Double, GreaterEqualOne]
+  type MemoryRetryMultiplier = Closed[W.`1.0`.T, W.`99.0`.T]
```

Comment:
Very cool!

I might add in a comment from refinement about why it's ≥ 1 not > 1:

We want to let people disable more-memory retry by adjusting the value down to 1. More convenient than adding/removing the entire parameter.

Reply:
Good point! I will add a comment here.